sources/packages/api/src/services/application/application.service.ts

```diff
@@ -214,10 +214,17 @@ export class ApplicationService extends RecordDataModelService<Application> {
       [],
       associatedFiles,
     );
-    application.creator = auditUser;
-    application.studentAssessments = [originalAssessment];
-    application.currentAssessment = originalAssessment;
-    await this.repo.save([application, newApplication]);
+    await this.connection.transaction(async (transactionalEntityManager) => {
+      await transactionalEntityManager
```

Comment:
Please assign `transactionalEntityManager.getRepository(Application)` to a const to avoid calling it twice.